llvm/utils/git/github-automation.py

```diff
@@ -603,12 +603,12 @@ def execute_command(self) -> bool:
         """
         This function reads lines from STDIN and executes the first command
         that it finds.  The 2 supported commands are:
-        /cherry-pick commit0 <commit1> <commit2> <...>
-        /branch <owner>/<repo>/<branch>
+        /cherry-pick< ><:> commit0 <commit1> <commit2> <...>
+        /branch< ><:> <owner>/<repo>/<branch>
```

Comment:
The branch command no longer exists.

Reply:
I see. Do we need the regex support for /cherry-pick command?